dd-sdk-android/src/test/kotlin/com/datadog/android/tracing/internal/domain/event/SpanEventSerializerTest.kt

```diff
@@ -106,6 +108,34 @@ internal class SpanEventSerializerTest {
         }
     }
 
+    @Test
+    fun `M sanitise the span metrics keys W level deeper than 9`(
```

Comment:
How do you ensure that the depth is greater than 9 in this test?

Reply:
Actually we just test that the `datadogConstraints` is being called with the right prefix. I guess I should rephrase this test message to express that instead as you have to know the implementation to deduct that calling it with a prefix `a` it will only allow keys up to 9 levels.